projects/canopy/src/lib/promo-card/promo-card-list/promo-card-list.component.html

```diff
@@ -0,0 +1,7 @@
+<lg-separator class="lg-promo-card-list__separator" variant="dotted"></lg-separator>
```

Comment:
I wondered if the separator should be part of this list component? Or if we should rely on people adding it manually as a sibling to the promo-card-list.

Reply:
I think it should be added manually as some people may not want to have it there.

Reply:
Yes I agree, we could allow to be added via a content projection slot, similar to the title and cards below.